### ISSUE TITLE
[Package Request] - dovecot

### ISSUE THREAD

**Issue by adelton** [amazonlinux/amazon-linux-2023#70]
**What package is missing from Amazon Linux 2022? Please describe and include package name.**

dovecot - IMAP and POP3 server - http://www.dovecot.org/

**Is this an update to existing package or new package request?**

New package in 2022.

**Is this package available in Amazon Linux 2?**

Yes.

**Number of users impacted**

No idea.

**Comment by lehrian** [comment]
This is seriously a crucial missing package. There are no replacements for an IMAP/POP server, and I would think a large number of users use AMI1/2 for mail servers. This will prevent many people from migrating to AL2023.

**Comment by VanDogh** [comment]
Agreed.  This is a show-stopper for me unless somebody can identify an equivalent package.  I don't understand at all why this would be dropped in AL2023.

**Comment by donikatz** [comment]
+1 showstopper. I really don't understand some of these decisions re common packages removed in AL2023.

**Comment by stewartsmith** [comment]
> +1 showstopper. I really don't understand some of these decisions re common packages removed in AL2023.

We did look at download statistics while building the first versions of Amazon Linux 2023, and very consciously started small, choosing to grow in the exact ways that customer requests would come in, and use it as an opportunity to further verify the need for each package we ship. We're happy to re-examine every package that's requested.

For Dovecot, would your expectations be that we package each new v2.3.x version as they come in?
If there's a v2.4, would you expect us to move up to it at the next point release of Amazon Linux, or next major version?

**Comment by lehrian** [comment]
> For Dovecot, would your expectations be that we package each new v2.3.x version as they come in?

Yes. 

> If there's a v2.4, would you expect us to move up to it at the next point release of Amazon Linux, or next major version?

I would want it at the next point release if possible. 

If that update cadence is too much, then anything would be better than nothing at this point.

**Comment by VanDogh** [comment]
I'd probably expect to stay on 2.3.x until AL2025 comes out, although it's not clear if there will be any more 2.3.x releases after the most recent one. since I think they are pushing on 2.4 now.  I'd be OK with either.  Thanks for giving this some attention.

**Comment by nmeyerhans** [comment]
Another question is about optional features. For example, the dovecot lucene FTS plugin ends up requiring a decent amount of additional dependencies that we don't currently provide.  Would dovecot without this plugin be useful, or is that a requirement?

**Comment by adelton** [comment]
Dovecot lucene FTS plugin is not a requirement, at least in the context of this issue and this package request.

**Comment by VanDogh** [comment]
No requirement for the Lucene FTS plugin, which is no longer being maintained.

**Comment by nmeyerhans** [comment]
Thanks for the feedback, everyone. I don't have an ETA, but we will include Dovecot in a forthcoming AL2023 update.  Please stand by. We'll note it in the release notes and resolve this issue when it's available.

**Comment by nmeyerhans** [comment]
Dovecot is available in the 20230419 repository version.

```plain
bash-5.2# cat /etc/os-release 
NAME="Amazon Linux"
VERSION="2023"
ID="amzn"
ID_LIKE="fedora"
VERSION_ID="2023"
PLATFORM_ID="platform:al2023"
PRETTY_NAME="Amazon Linux 2023"
ANSI_COLOR="0;33"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2023"
HOME_URL="https://aws.amazon.com/linux/"
BUG_REPORT_URL="https://github.com/amazonlinux/amazon-linux-2023"
SUPPORT_END="2028-03-01"
bash-5.2# rpm -q system-release
system-release-2023.0.20230419-0.amzn2023.noarch
bash-5.2# dnf list dovecot
Last metadata expiration check: 0:00:15 ago on Thu Apr 20 17:02:30 2023.
Available Packages
dovecot.x86_64                                              1:2.3.20-1.amzn2023.0.1                                              amazonlinux
```

**Comment by adelton** [comment]
Thank you.
